cypress.config.js

```diff
@@ -15,7 +15,7 @@ module.exports = defineConfig({
           const randomNumber = Math.ceil(Math.random(1000) * 1000);
           return {
             username: faker.name.firstName() + `${randomNumber}`,
```

Comment:
Consider using template literals to construct the username string for better readability and performance.